README.md

```diff
@@ -27,7 +27,6 @@ See: https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/blacklovertech"><img src="https://avatars.githubusercontent.com/u/93751501?v=4?s=100" width="100px;" alt="Janarthanan"/><br /><sub><b>Janarthanan</b></sub></a><br /><a href="https://github.com/tungbq/aws-lab-with-terraform/commits?author=blacklovertech" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="https://github.com/Abraham12611"><img src="https://avatars.githubusercontent.com/u/113689862?v=4?s=100" width="100px;" alt="Abraham Dahunsi"/><br /><sub><b>Abraham Dahunsi</b></sub></a><br /><a href="https://github.com/tungbq/aws-lab-with-terraform/commits?author=Abraham12611" title="Code">💻</a></td>
       <td align="center" valign="top" width="14.28%"><a href="http://devazc.netlify.app"><img src="https://avatars.githubusercontent.com/u/122866331?v=4?s=100" width="100px;" alt="Deva Chandragiri"/><br /><sub><b>Deva Chandragiri</b></sub></a><br /><a href="https://github.com/tungbq/aws-lab-with-terraform/commits?author=Devazc" title="Code">💻</a></td>
-      <td align="center" valign="top" width="14.28%"><a href="https://vedangithokal.hashnode.dev"><img src="https://avatars.githubusercontent.com/u/117835144?v=4?s=100" width="100px;" alt="Vedangi Thokal"/><br /><sub><b>Vedangi Thokal</b></sub></a><br /><a href="https://github.com/tungbq/aws-lab-with-terraform/commits?author=vedangit" title="Code">💻</a></td>
```

Comment:
Please keep this line as is.

Reply:
BTW, should i remove the above line ??

Reply:
No we should keep it, the idea is to keep everything latest with main. And only change the thing related to the issue
We need to rebase the branch first I believe